File: backend/OdevBackend/service/src/main/java/com/example/service/model/PaymentType.java
package com.example.service.model;

public enum PaymentType {
    CREDIT_CARD("Credit Card"),
    PAYWallet("PayWallet");

    private final String displayName;

    PaymentType(String displayName) {
        this.displayName = displayName;
    }

    public String getDisplayName() {
        return displayName;
    }
}

File: mobil/lib/models/PaymentType.dart
enum PaymentType {
  CREDIT_CARD,
  PAYWallet,
}

extension PaymentTypeExtension on PaymentType {
  String get displayName {
    switch (this) {
      case PaymentType.CREDIT_CARD:
        return "Credit Card";
      case PaymentType.PAYWallet:
        return "Pay Wallet";
      default:
        return '';
    }
  }
}